Sources/AppStoreConnectCLI/Services/Operations/InviteTesterOperation.swift

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct InviteTesterOperation: APIOperation {
+
+    struct InviteBetaTesterDependencies {
+        let betaTesterResponse: (APIEndpoint<BetaTesterResponse>) -> Future<BetaTesterResponse, Error>
+    }
+
+    private let options: InviteBetaTesterOptions
+
+    init(options: InviteBetaTesterOptions) {
+        self.options = options
+    }
+
+    func execute(with dependencies: InviteBetaTesterDependencies) -> AnyPublisher<BetaTester, Error> {
+
+        let requests = options.betaGroupIds.map { (id: String) -> AnyPublisher<BetaTesterResponse, Error> in
+            let endpoint = APIEndpoint.create(
+                betaTesterWithEmail: options.email,
+                firstName: options.firstName,
+                lastName: options.lastName,
+                betaGroupIds: [id]
+            )
+
+            return dependencies
+                .betaTesterResponse(endpoint)
+                .eraseToAnyPublisher()
+        }
+
+        let testerId = try! Publishers.ConcatenateMany(requests)
```

Comment:
We should avoid `try!` and throw our errors inside of our publisher transforms or throw them from the function itself